Updated Google Calendar Crawler- Changed Nomenclature and token type

rberry206 opened the pull request:
Added comments to the calendar crawler

Commit:
Updated Google Calendar Crawler

Added comments to the calendar crawler

Commit:
Updated for Travis CI

mrvirus9898 commented on scripts/googleCalendarCrawler/calendarCrawler.py:
```diff
@@ -1,50 +1,77 @@
     # Call the Calendar API
     now = datetime.datetime.utcnow().isoformat() + 'Z' # 'Z' indicates UTC time
-    print('Getting the upcoming 10 events')
+    print('Getting the upcoming 100 events')
```
Can we get a current Timestamp in the log so we know when this code executes, and how long it takes to execute?

MikeJLeon commented:
Seems to work, only caveat is the requirement of the credentials.json. we'll need to look for a solution to that, which i'm sure secrets manager on aws will handle.

MikeJLeon reviewed (approved):
Seems to work, only caveat is the requirement of the credentials.json. we'll need to look for a solution to that, which i'm sure secrets manager on aws will handle.

Commit:
Updated Google Calendar Crawler

Updated crawler with proper nomenclature. Changed json token to pickle. Does *not* write to a database yet. Credentials and Token will exist in Secrets Manager.
